server/src/main/webapp/WEB-INF/rails/webpack/views/pages/config_repos/modals.tsx

```diff
@@ -39,18 +43,21 @@ import materialStyles from "./materials.scss";
 
 type EditableMaterial = SaveOperation
   & { repo: ConfigRepo }
+  & { userProps: EntriesVM }
   & { isNew: boolean }
   & RequiresPluginInfos
   & { error?: m.Children }
   & { resourceAutocompleteHelper: Map<string, string[]> };
 
 class MaterialEditWidget extends MithrilViewComponent<EditableMaterial> {
   view(vnode: m.Vnode<EditableMaterial>) {
-    const pluginList = _.map(vnode.attrs.pluginInfos(), (pluginInfo: PluginInfo) => {
+    const { repo, userProps, isNew, error, pluginInfos, resourceAutocompleteHelper } = vnode.attrs;
```

Comment:
No functional changes here, just tidying the subsequent code to make it a little more readable with less `vnode.attrs` noise.